docs/src/start/connect-aws.md

```diff
@@ -49,83 +69,180 @@ You can then check the content of that certificate.
 
-```sh
-tedge config get device.id
-```
+3. Click **Create policy**, and enter a policy name (e.g. `thin-edge.io`)
+
+4. Under the **Policy statements** tab click on ***JSON*** and add the following JSON:
+
+    :::tip
+    If you haven't provided the [User Context](#user-context) then it is strongly encouraged to do so, as the following snippet will be updated to reflect your AWS IoT Core settings.
+    :::
+
+    <UserContext>
+
+    ```json
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Effect": "Allow",
+          "Action": "iot:Connect",
+          "Resource": "arn:aws:iot:$AWS_REGION:$AWS_ACCOUNT_ID:client/${iot:Connection.Thing.ThingName}"
+        },
+        {
+          "Effect": "Allow",
+          "Action": "iot:Subscribe",
+          "Resource": [
+            "arn:aws:iot:$AWS_REGION:$AWS_ACCOUNT_ID:topicfilter/thinedge/${iot:Connection.Thing.ThingName}/cmd/#",
+            "arn:aws:iot:$AWS_REGION:$AWS_ACCOUNT_ID:topicfilter/$aws/things/${iot:Connection.Thing.ThingName}/shadow/#",
+            "arn:aws:iot:$AWS_REGION:$AWS_ACCOUNT_ID:topicfilter/thinedge/devices/${iot:Connection.Thing.ThingName}/test-connection"
+          ]
+        },
+        {
+          "Effect": "Allow",
+          "Action": "iot:Receive",
+          "Resource": [
+            "arn:aws:iot:$AWS_REGION:$AWS_ACCOUNT_ID:topic/thinedge/${iot:Connection.Thing.ThingName}/cmd",
+            "arn:aws:iot:$AWS_REGION:$AWS_ACCOUNT_ID:topic/thinedge/${iot:Connection.Thing.ThingName}/cmd/*",
+            "arn:aws:iot:$AWS_REGION:$AWS_ACCOUNT_ID:topic/$aws/things/${iot:Connection.Thing.ThingName}/shadow",
+            "arn:aws:iot:$AWS_REGION:$AWS_ACCOUNT_ID:topic/$aws/things/${iot:Connection.Thing.ThingName}/shadow/*",
+            "arn:aws:iot:$AWS_REGION:$AWS_ACCOUNT_ID:topic/thinedge/devices/${iot:Connection.Thing.ThingName}/test-connection"
+          ]
+        },
+        {
+          "Effect": "Allow",
+          "Action": "iot:Publish",
+          "Resource": [
+            "arn:aws:iot:$AWS_REGION:$AWS_ACCOUNT_ID:topic/thinedge/${iot:Connection.Thing.ThingName}/td",
+            "arn:aws:iot:$AWS_REGION:$AWS_ACCOUNT_ID:topic/thinedge/${iot:Connection.Thing.ThingName}/td/*",
+            "arn:aws:iot:$AWS_REGION:$AWS_ACCOUNT_ID:topic/$aws/things/${iot:Connection.Thing.ThingName}/shadow",
+            "arn:aws:iot:$AWS_REGION:$AWS_ACCOUNT_ID:topic/$aws/things/${iot:Connection.Thing.ThingName}/shadow/*",
+            "arn:aws:iot:$AWS_REGION:$AWS_ACCOUNT_ID:topic/thinedge/devices/${iot:Connection.Thing.ThingName}/test-connection"
+          ]
+        }
+      ]
+    }
+    ```
+
+    </UserContext>
+
+    :::note
+    A static form of the example policy can be downloaded from the [repository](https://raw.githubusercontent.com/thin-edge/thin-edge.io/main/docs/src/start/aws-example-policy.json), though you will need to replace the `<region>` and `<account-id>` placeholders with your AWS IoT Core values.
```

Comment:
For some reasons, the link is not really visible (because inside a note?).

```suggestion
    A static form of the example policy can be downloaded from the [__repository__](https://raw.githubusercontent.com/thin-edge/thin-edge.io/main/docs/src/start/aws-example-policy.json), though you will need to replace the `<region>` and `<account-id>` placeholders with your AWS IoT Core values.
```

Reply:
I switched from a "note" to an "info" block, is this more obvious here the link is?

![image](https://github.com/user-attachments/assets/384ecafe-5483-4ac4-ab62-113e6d07867f)

Reply:
Slight improvement, but that's okay.